packages/grafana-ui/src/utils/standardEditors.tsx

```diff
@@ -1,320 +1,17 @@
-import React from 'react';
-
-import {
-  DataLink,
-  dataLinksOverrideProcessor,
-  FieldConfigPropertyItem,
-  FieldType,
-  NumberFieldConfigSettings,
-  numberOverrideProcessor,
-  standardEditorsRegistry,
-  StandardEditorsRegistryItem,
-  StringFieldConfigSettings,
-  stringOverrideProcessor,
-  identityOverrideProcessor,
-  TimeZone,
-  FieldColor,
-  FieldColorConfigSettings,
-  StatsPickerConfigSettings,
-  displayNameOverrideProcessor,
-  FieldNamePickerConfigSettings,
-} from '@grafana/data';
-
-import {
-  NumberValueEditor,
-  SliderValueEditor,
-  RadioButtonGroup,
-  StringValueEditor,
-  StringArrayEditor,
-  SelectValueEditor,
-  MultiSelectValueEditor,
-  TimeZonePicker,
-} from '../components';
-import { FieldNamePicker } from '../components/MatchersUI/FieldNamePicker';
-import { ColorValueEditor } from '../components/OptionsUI/color';
-import { FieldColorEditor } from '../components/OptionsUI/fieldColor';
-import { DataLinksValueEditor } from '../components/OptionsUI/links';
-import { StatsPickerEditor } from '../components/OptionsUI/stats';
-import { UnitValueEditor } from '../components/OptionsUI/units';
-import { Switch } from '../components/Switch/Switch';
-
 /**
  * Returns collection of common field config properties definitions
+ *
+ * @deprecated options editors are managed by grafana core
  */
 export const getStandardFieldConfigs = () => {
```

Comment:
I am not convinced that "deprecating" this is a good path.  The API is not really useable directly, so 🤷

Reply:
I updated the PR so we just remove them